README.md

```diff
@@ -34,9 +34,9 @@ const combobo = new Combobo();
   * Defaults to `.combobox`
 * `list` (_HTMLElement|String_): The selector for the list element or the list element reference.
   * Defaults to `.listbox`
-* `options` (_HTMLElement|String_): The selector for the option elements (qualified within the list) or the option element references.
+* `options` (_Array|String_): An array of the selector(s) for the option elements (qualified within the list) or the option element references.
```

Comment:
this should be:
An array of HTMLElements or a string selector (to be qualified within the list element)